docs/custom-account-fields.md

```diff
@@ -0,0 +1,182 @@
+# Account custom fields
+
+This document shows you how to setup account custom fields for your galoy instance.
+
+## Configuration
+
+```
+customFields:
+  - name: "customFieldName"
+    type: <"string" | "integer" | "float" | "boolean"> # optional (default: "string")
+    defaultValue: <value> # optional
+    editable: <true | false> # optional (default: false) - true if field can be updated by the user
+    required: <true | false> # optional (default: false) - true if field is required by mutations
+    index: <true | false> # optional (default: false) - true if field should have an index in db
+    unique: <true | false> # optional (default: false) - true if field should have a unique index in db
+```
+
+## Example
+
+Add the next configuration to `default.yaml`
+```
+accounts:
+  ...
+  customFields:
+    - name: "nationalId"
+      type: "integer"
+      defaultValue: 0
```

Comment:
isn't it a use case where `index` and `unique` should be true, and not have a defaultValue?